src/js/api/local-storage-api.js

```diff
@@ -196,7 +193,7 @@ export default class LocalStorage {
         // 3. сохранить сторедж
         // вернуть результат
     }
-    addToRead(newReaded) {
+    addToReaded(newReaded) {
```

Comment:
Має бути насправді newRead, але я не знаю, де воно ще може використовуватися. Але треба буде змінити.
```suggestion
    addToRead(newReaded) {
```